torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -419,6 +419,30 @@ def pad_bounding_box(
 crop_image_pil = _FP.crop
 
 
+def crop_bounding_box(
+    bounding_box: torch.Tensor,
+    format: features.BoundingBoxFormat,
+    image_size: Tuple[int, int],
+    top: int,
+    left: int,
+    height: int,
+    width: int,
```

Comment:
As an discussion opener: since these are unused, should we have them on the kernel at all?

Reply:
I somehow missed this comment. This is a fair remark though I'm split on what we should do.

On one hand, passing unnecessary parameters creates noisy method signatures. On the other, the current proposal aligns the image kernel with the bbox in terms of parameters which is nice.

Assume we actually adopt the idea of putting kernel dispatcher on the object level (Image, Bbox, Mask). Then all these methods need to receive the same arguments (excluding meta-data that are provided directly from the class). But even in this case, the low-level kernels don't have to receive these parameters. I would be in favour of removing them from here but I want to hear both your thoughts.

Reply:
Yes, makes sense. Let's remove unused args